homework/index.js

```diff
@@ -96,6 +96,12 @@
   }
 
   function deleteContainers() {
+    const selectButton = document.getElementById('selectButton');
+    let firstOption = selectButton.firstChild;
+    while (firstOption) {
+      selectButton.removeChild(firstOption);
+      firstOption = selectButton.firstChild;
+    }
```

Comment:
This is an inefficient way to solve the problem of an ever-growing list of option elements. Basically, you are re-creating each time the very same option elements, each time the user changes the selection. To avoid growing the existing list you erase the previous content. The proper solution would be to create these option elements **once only**, in your main function after fetching the repo data, sorting it and creating the select element.